chalice/deploy/deployer.py

```diff
@@ -559,9 +581,10 @@ def _build_lambda_function(self,
             config.app_name, config.chalice_stage, name)
         security_group_ids = config.security_group_ids
         subnet_ids = config.subnet_ids
-        if security_group_ids is None or subnet_ids is None:
+        if security_group_ids is None and subnet_ids is None:
```

Comment:
Is this if statement still needed? It looks like it gets handled in `_get_vpc_params` and is not necessarily needed.